src/WebJobs.Extensions.DurableTask/RetryOptions.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using DurableTaskCore = DurableTask.Core;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
```

Comment:
Up until this point, all public types that are intended to be used by user code went into the `Microsoft.Azure.WebJobs` namespace. This is what I observed in other extensions as well. I think we should do the same here.